election/views.py

```diff
@@ -86,3 +96,52 @@ def create(self, request, *args, **kwargs):
             return Response(status=status.HTTP_400_BAD_REQUEST)
         headers = self.get_success_headers(serializer.data)
         return Response(status=status.HTTP_201_CREATED, headers=headers)
+
+
+
+
+class ResultAPIView(APIView):
+    """ 
+    View to list the result of an election using majority judgment.
+    """
+
+
+    def create(self, request, *args, **kwargs):
+
+        election = request.data.get("election", "")
```

Comment:
Get the election id through the GET parameters prevent DRF to automagically document the API. Look at `views.py/ElectionDetailsAPIView` and `serializers.py/ElectionViewSerializer` to have an example of how to use it properly.